src/lib/components/progress/QLinearProgress.svelte

```diff
@@ -13,23 +13,25 @@
     borderRadius: rounded ? "12px" : "0",
   };
 
-  $: containerClasses = createClasses(["small-space", "border"], {
-    component: "q-linear-progress",
-    userClasses,
-  });
-
-  $: progressClasses = createClasses([from], {
-    component: "q-linear-progress",
-    element: "progress",
-  });
-
   $: containerStyle = createStyles(roundedStyle, userStyles);
   $: progressStyle =
     from === "right"
       ? `clip-path: polygon(100% 0%, 100% 100%, ${100 - value}% 100%, ${100 - value}% 0%);`
       : `clip-path: polygon(0% 0%, 0% 100%, ${value}% 100%, ${value}% 0%);`;
 </script>
 
-<div class={containerClasses} style={containerStyle} {...$$restProps}>
-  <div class={progressClasses} style={progressStyle} />
+<div
+  class="q-linear-progress {userClasses}"
+  style={containerStyle}
```

Comment:
Probably better to have `style:border-radius={rounded ? "12px" : "0"}` instead of using `createStyles`. That would also allow us to remove the `userStyles` prop